test/development/acceptance-app/error-recovery.test.ts

```diff
@@ -302,7 +302,7 @@ describe('Error recovery app', () => {
            "Index.useCallback[increment] index.js (7:11)",
            "button <anonymous>",
            "Index index.js (12:7)",
-           "Page index.js (8:16)",
+           "Page index.js (10:5)",
```

Comment:
i don't really understand this, but given the comment i guess it makes sense, the array layout triggers swc formatter to use more newlines so the symbol is incorrectly bumped down?